packages/contract-wrappers/test/exchange_wrapper_test.ts

```diff
@@ -368,6 +368,23 @@ describe('ExchangeWrapper', () => {
             await web3Wrapper.awaitTransactionSuccessAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
             isPreSigned = await contractWrappers.exchange.isPreSignedAsync(hash, signerAddress);
             expect(isPreSigned).to.be.true();
+
+            const preSignedSignature = '0x06';
+            const isValidSignature = await contractWrappers.exchange.isValidSignatureAsync(
+                hash,
+                signerAddress,
+                preSignedSignature,
+            );
+            expect(isValidSignature).to.be.true();
+
+            // Test our TS implementation of signature validation
+            const isValidSignatureInTs = await signatureUtils.isValidSignatureAsync(
```

Comment:
If our TS implementation doesn't live in that package - why do we test it here?

Reply:
Because we don't do any "integration tests" in OrderUtils and for this, it needs to be an integration test. Agree it's weird, but this package is gone after "better revert reasons" merges.